source/about/out-of-band-collaboration.rst

```diff
@@ -0,0 +1,66 @@
+Out-of-Band collaboration
+==========================
+
+Out-of-Band (OOB) collaboration is a method of communication that occurs outside an organization’s primary business systems on a separate communications channel. 
+
+This approach ensures teams can communicate and coordinate even when their main networks are compromised. Learn more about `key use cases <#key-use-cases>`__ below, and visit the Mattermost blog to learn how OOB collaboration helps you `maintain business continuity <https://mattermost.com/blog/out-of-band-communication-maintaining-business-continuity/>`_.
+
+Mattermost for OOB
+------------------
+
+Organizations can use Mattermost as an OOB solution by either self-hosting their instance or leveraging Mattermost’s Cloud SaaS offering.  
+
+Self-hosted
+~~~~~~~~~~~~
+
+We recommend one of the following self-hosted deployment options:
+
+- `Azure Marketplace Offering <http://mattermost-docs-preview-pulls.s3-website-us-east-1.amazonaws.com/7816/deploy/server/deploy-kubernetes.html>`__: Use a Kubernetes deployment option available on the Azure Marketplace, separate from your own infrastructure.
+
+- `Self-Hosted Kubernetes Deployment <http://mattermost-docs-preview-pulls.s3-website-us-east-1.amazonaws.com/7816/deploy/server/deploy-kubernetes.html>`__: Deploy Mattermost on Kubernetes in your own separate infrastructure or cloud, ensuring full control and customization.
+
+Cloud
+~~~~~
+
+If you’d prefer a turnkey out-of-band solution that requires minimal IT overhead, we recommend Mattermost’s :doc:`Cloud SaaS offering </about/cloud-subscriptions>` for your out-of-band solution. Mattermost can provision and manage dedicated infrastructure for your instance, and Mattermost manages the SaaS platform within your environment.  
```

Comment:
> and Mattermost manages the SaaS platform within your environment.

I think this still may be a bit misleading as we don't offer running the SAAS platform entirely. We just offer ways to deploy and manage Mattermost itself.